core/templates/dev/head/app.js

```diff
@@ -380,6 +380,13 @@ oppia.config(['toastrConfig', function(toastrConfig) {
   });
 }]);
 
+oppia.config(['recorderServiceProvider', function(recorderServiceProvider) {
```

Comment:
What is this for?

Reply:
I think this is some merge issue, I didn't add this code.

Reply:
This is indeed a merge issue -- try merging develop into your branch, @nithusha21.